husky-fhir/husky-fhir-emed-ch/husky-fhir-emed-ch-epr/src/main/java/org/projecthusky/fhir/emed/ch/epr/enums/RegularUnitCodeAmbu.java

```diff
@@ -660,6 +659,19 @@ public enum RegularUnitCodeAmbu implements ValueSetEnumInterface {
        "nl",
        "nl",
        "nl"),
+    /**
+     * EN: nmol.<br>
+     * DE: nmol.<br>
+     * FR: nmol.<br>
+     * IT: nmol.<br>
+     */
+    NMOL("nmol",
+        "http://unitsofmeasure.org",
+        "nanomole",
+        "nmol",
+        "nmol",
+        "nmol",
+        "nmol"),
```

Comment:
Sorry, I failed to update this enum while updating the value sets. From https://github.com/project-husky/husky/commit/c029d01329b58e03171715e3c78fedc378612550#diff-41404c6413bed2afac45c3d235788fb6917739dac0a8b5051cf9f106c92134a5, it seems we are also missing APPLICATION_UNIT_OF_PRODUCT_USAGE_QUALIFIER_VALUE (413568008) here.

Reply:
In fact those are too new for the [current version of the CH EMED EPR value set](https://fhir.ch/ig/ch-emed-epr/ValueSet-ch-emed-epr-amount-quantity-unit-code.html)... I will remove NMOL from this enum and from the CdTyp9 conversion and will have to be added back with the next version of CH EMED EPR based on CH EMED 5.0.0. The enum units affected will be:

- [NMOL (new with CH EMED 5.0.0)](https://github.com/hl7ch/ch-emed/issues/250)
- [APPLICATION_UNIT_OF_PRODUCT_USAGE_QUALIFIER_VALUE (new with CH EMED 5.0.0)](https://github.com/hl7ch/ch-emed/issues/250)
- [PIECE (different system and code value with CH EMED 5.0.0)](https://github.com/hl7ch/ch-emed/issues/252)